### PR TITLE
Reworks some trader pools.

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

adds sol gov self assembling kit.
Adds box of labled random seeds.
handheld body scanner works faster now

Adjusts trader shuttle as follows:
Civ:
Removes:
Utility belt (tools are common) welding goggles (might just do adv goggles, cargo for sunglasses or get implant from rnd), magboots (Easy from rnd, use modules more) nanopaste (anyone who wants this got this from rnd)
Adds / tweaks:
Increase sol gov cigs weight (main export) Cham kit (for costume shenanigins), satchel flat (hiding trader goods, theming, storage), book of babel (rp, fun)

Materials:
no longer roll the same material twice, adding only 1 material to the pile.

Science:
Removes robotic brain (no one ever buys this / will need this since robotics). Slightly lowers research paper weight (we got rnd rework coming soon anyways)
Adds: Telescience console. (It's code green or blue, people haven't touched it in years, what could go wrong?) Surveillance disk, why not have the AI try to convince crew to buy something for it? This gives AI only hearing from camera, it DOESN'T give them malf picker.

Medical:
Removes: Lazurus bottle(omni bottle is there for that), IMS(science will have these by the time traders come along), the two viro bottles(either a healing virus is made or a virologist will not know how to use it).
Adds: medbeam(Very visable, probably not many antags. If there are, they will know and take it out), handheld body scanner(Not that useful but hey, on the go could be of use), syndicate sleeper board (a sleeper with diffrent chemicals and self service)

Security (could use more)
Removes lethal enforcer (rubber can take lethal bullets, weight increased) and combat shotgun(dual tube is better, dual tube increased) Clusterbang(never going to be used well outside blob)
Adds: Univsersal sol trader gun assembly kit, for security to fiddle with rnd guns (or others)

Engineering
Increased items to 8
Removes: cluster metal foam grenade(useless outside major breaches which make station red alert)
adds: advanced jetpack modules(miners are stingy), oil slick potions (speed, request from discord), atmos holosign creators

large item
adds: snow machine
adds: cereal machine

service:
removes mining voucher (miners have enough points generally) seeds (increased to box of seeds)
adds: that box of seeds, burger dispenser module (for mcdonalds chef), construct shell (for chaplain)

## Why It's Good For The Game

Freshens up the pool, makes it to more rare items / less common items, or other interesting things

## Testing

Compiles, icon works, pools more fresh

## Changelog
:cl:
tweak: Heavily adjusts the trader pool
tweak: handheld body scanner scans faster
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -556,6 +556,23 @@
 		B.reagents.add_reagent(reag, 30)
 		B.name = "[reag] bottle"
 
+/obj/item/storage/box/botany_labled_seeds
+	name = "botanist labled random seeds kit"
+	desc = "A box intended for experienced botanists"
+
+/obj/item/storage/box/botany_labled_seeds/populate_contents()
+	for(var/I in 1 to 7)
+		new /obj/item/seeds/random/labelled(src)
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/boxes.dm:565]:
```suggestion
/obj/item/storage/box/botany_labled_seeds
	name = "botanist labled random seeds kit"
	desc = "A box intended for experienced botanists."

/obj/item/storage/box/botany_labled_seeds/populate_contents()
	for(var/I in 1 to 7)
		new /obj/item/seeds/random/labelled(src)
```
"labled"? And the seeds path is written with 2 "l" too.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -556,6 +556,23 @@
...
+
+/obj/item/storage/box/telescience
+	name = "babies first telescience kit"
+	desc = "A now restricted kit for those who want to learn about telescience!"
+
+/obj/item/storage/box/telescience/populate_contents()
+	new /obj/item/circuitboard/telesci_pad(src)
+	new /obj/item/circuitboard/telesci_console(src)
+
+
```
Comment (Burzah) [inline code/game/objects/items/weapons/storage/boxes.dm:575]:
```suggestion
```

**Comment by Henri215** [comment]
The labled/labelled name remains

**Comment by Qwertytoforty** [comment]
> labelled

fixed, sorry

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -556,6 +556,22 @@
 		B.reagents.add_reagent(reag, 30)
 		B.name = "[reag] bottle"
 
+/obj/item/storage/box/botany_labled_seeds
+	name = "botanist lablled random seeds kit"
+	desc = "A box intended for experienced botanists."
+
+/obj/item/storage/box/botany_labled_seeds/populate_contents()
```
Comment (Henri215) [inline code/game/objects/items/weapons/storage/boxes.dm:563]:
```suggestion
/obj/item/storage/box/botany_labelled_seeds
	name = "botanist labelled random seeds kit"
	desc = "A box intended for experienced botanists."

/obj/item/storage/box/botany_labelled_seeds/populate_contents()
```

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 4]
```diff
@@ -378,13 +381,12 @@
...
 	loot = list(
 				// Mining
-				/obj/item/mining_voucher = 100,
 				/obj/item/pickaxe/drill/jackhammer = 100,
 				/obj/item/gun/energy/kinetic_accelerator/experimental = 100,
 				/obj/item/borg/upgrade/modkit/aoe/turfs/andmobs = 100,
 
 				// Botanist
-				/obj/item/seeds/random/labelled = 100,
+				/obj/item/storage/box/botany_labled_seeds = 100,
```
Comment (Henri215) [inline code/game/objects/effects/spawners/lootdrop.dm:389]:
```suggestion
				/obj/item/storage/box/botany_labelled_seeds = 100,
```

File: code/modules/crafting/guncrafting.dm [hunk 5]
```diff
@@ -104,6 +104,10 @@
 	desc = "A suitcase containing the necessary gun parts to build a full gun, when combined with a gun kit. Use it directly on a gunkit to rapidly assemble it."
 	icon_state = "syndicase"
 
+/obj/item/weaponcrafting/gunkit/universal_gun_kit/sol_gov
+	name = "\improper sol gov universal self assembling gun parts kit"
+	icon_state = "totallynotikea"
```
Comment (Henri215) [inline code/modules/crafting/guncrafting.dm:109]: totally not ikea?

Comment (Qwertytoforty) [reply code/modules/crafting/guncrafting.dm:109]: colouring of the self assembling kit is the colour of the ikea colour pallet. Can be changed

Comment (Henri215) [reply code/modules/crafting/guncrafting.dm:109]: Discovering new colors every day 0-0
I would change it to "solgovcase" and turn this joke into a comment.

Comment (Qwertytoforty) [reply code/modules/crafting/guncrafting.dm]: renamed to solcase

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**